sonar-php-plugin/src/test/java/org/sonar/plugins/php/phpunit/PhpUnitResultParserTest.java

```diff
@@ -117,6 +117,16 @@ public void shouldGenerateTestsMeasures() {
     verify(context).saveMeasure(bananaFile, CoreMetrics.TEST_EXECUTION_TIME, 570.0);
   }
 
+  /**
+   * Should not fail if system output is found
+   */
+  @Test
+  public void shouldNotFailIfSystemOutputIsFound() {
+    parser.parse(TestUtils.getResource(MockUtils.PHPUNIT_REPORT_DIR + "phpunit.systemout.xml"));
+
+    verify(context, never()).saveMeasure(any(org.sonar.api.resources.File.class), any(Metric.class), anyDouble());
```

Comment:
@kinow It's weird to test that everything is ok by checking that's no measures are saved. We use this assertion to test cases when some problem appear. As already said I think it will be enough to add "new" tags to existing xml test file with no changes to PhpUnitResultParserTest.java